protocol/typeSerialization.hpp

```diff
@@ -281,6 +281,13 @@ namespace protocol
         return (a << 64) | b;
     }
 
+    constexpr void addUUID(std::vector<uint8_t> &out, const __int128 &data)
+    {
+        for (uint8_t shift_value = 120; shift_value >= 0; shift_value -= 8) {
+            out.push_back((data >> shift_value) && 0xFF);
+        }
+    }
```

Comment:
Why not just use addLong two times ?
#48 also adds this function so once again, I think waiting for merge is the good move.